app/assets/javascripts/miq_application.js

```diff
@@ -1522,7 +1522,7 @@ function miqToolbarOnClick(_e) {
   // put url_parms into params var, if defined
   var params;
   if (button.data("url_parms")) {
-    if (button.data('url_parms').match("_div$")) {
+    if (button.data("send_checked")) {
```

Comment:
..except that `params = miqSerializeForm(button.data('url_parms'))` should happen when `url_parms` but not `send_checked`.